Firestore/core/test/firebase/firestore/util/status_test.cc

```diff
@@ -113,6 +113,18 @@ TEST(Status, FromErrno) {
           "Already exists: Cannot write file \\(errno .*: File exists\\)"));
 }
 
+TEST(Status, Annotate) {
+  Status a(FirestoreErrorCode::NotFound, "fork not found");
+  Status b = a.Annotate("could not eat dinner");
+  ASSERT_EQ(FirestoreErrorCode::NotFound, b.code());
+  ASSERT_EQ("fork not found; could not eat dinner", b.error_message());
```

Comment:
You're asserting kinda unrelated things. Or at least, things that could fail separately for different reasons. You could change these to expect. Optional.

Reply:
Will do in the next PR.